src/app/components/Buttons/SearchButton.jsx

```diff
@@ -1,23 +1,28 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import SearchIconReversed from './SearchIconReversed';
+
+import {
+  Button,
+  ButtonTypes,
+  Icon,
+  IconNames,
+} from '@nypl/design-system-react-components';
 
 const SearchButton = ({
   id,
-  className,
   onClick,
   value,
 }) => (
-  <button
+  <Button
     id={id}
-    className={`${className}`}
-    onClick={onClick}
+    buttonType={ButtonTypes.Primary}
     type="submit"
     aria-controls="results-description"
+    onClick={onClick}
   >
+    <Icon name={IconNames.search} decorative />
```

Comment:
I made a PR in DS to fix how the icon looks in the search button: https://github.com/NYPL/nypl-design-system/pull/520/files#diff-075a2ceff5682f91348cc586ee6011c51b85ed1ef742c45a992d48d6fdf34386

Try updating to this.